src/main/java/org/apache/pirk/responder/wideskies/ResponderService.java

```diff
@@ -0,0 +1,73 @@
+package org.apache.pirk.responder.wideskies;
```

Comment:
Just move the license comment to the top of the file, to fit with the style adopted throughout.